lms/djangoapps/instructor/enrollment.py

```diff
@@ -158,7 +160,10 @@ def enroll_email(course_id, student_email, auto_enroll=False, email_students=Fal
             email_params['message_type'] = 'enrolled_enroll'
             email_params['email_address'] = student_email
             email_params['full_name'] = previous_state.full_name
-            send_mail_to_student(student_email, email_params, language=language)
+            if is_testing_environment():
+                send_mail_to_student(student_email, email_params, language=language)
+            else:
+                compose_and_send_adg_course_enrollment_invitation_email(student_email, email_params)
```

Comment:
Enrollment email is also sent from instructor APIs `edx-platform/lms/djangoapps/instructor/views/api.py:599`. I am not sure if it is in the scope.

Reply:
This is for inviting users by emails to some specific course, if the user is not on. our platform. The code you suggested create and enroll user in course which is slightly different functionality.